storybookjs/storybook
# Log network location of Storybook app on startup

Issue:
Fixes #4231 

## What I did

This change logs an additional network-reachable url for the Storybook app:
<img width="425" alt="screen shot 2018-09-25 at 8 51 56 pm" src="https://user-images.githubusercontent.com/471731/46056318-e1601300-c104-11e8-8785-726a4eb59e2b.png">


## How to test

Is this testable with Jest or Chromatic screenshots? Not really
Does this need a new example in the kitchen sink apps? No
Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
